middle_end/flambda2/terms/effects_and_coeffects.mli

```diff
@@ -15,7 +15,7 @@
 (* Effects and coeffects *)
 
 (** A pair of an effect and a coeffect. *)
```

Comment:
Seems like most/all of the comments in this file should be updated.

Reply:
done in 5f0ffc47c0c4f8e5d9f9009911763b7c7dcd9950